lib/SetAPI/generic/constants.py

```diff
@@ -1,3 +1,39 @@
+"""Various strings used throughout the repo."""
+
 INC_ITEM_INFO = "include_item_info"
 INC_ITEM_REF_PATHS = "include_set_item_ref_paths"
 REF_PATH_TO_SET = "ref_path_to_set"
+
+
+ASSEMBLY = "assembly"
```

Comment:
added a bunch of constants for strings that are either frequently used or long enough that they are a nuisance to have to write out